apps/sudoku-frontend/src/app/pages/classic-game/classic-game.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ClassicGameComponent } from './classic-game.component';
+
+describe('ClassicGameComponent', () => {
+  let component: ClassicGameComponent;
+  let fixture: ComponentFixture<ClassicGameComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ClassicGameComponent]
```

Comment:
I schätz mal bei dem MockModule fehlen noch die Imports deswegen gehen die Unit tests ned

Reply:
@waghD @vambi 
Könnte einer von euch sich das mit dem failed Test bitte noch anschaun, konnte es leider jetzt auf die schnelle nicht lösen.

Anschließend würde ich vorschlagen, dass wir diesen Merge Request schließen um es für den Showcase morgen während der LVA auf dem Main branch zu haben.

Da auf dieser Seite noch sekundäre Funktionalität wie Buttons zum speichern, notizen etc. fehlt, würde ich vorschlagen weitere nicht-zeitkritische Änderungen dann in Verbindung mit diesem Issue #32 einzuarbeiten.

Reply:
Werd mi drum kümmern